tickit/devices/eiger/data/dummydata.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from tickit.devices.eiger.data.image import Image, deduce_encoding
+
+_DUMMY_IMAGE_BLOBS: List[bytes] = []
+
+
+def dummy_image(index: int) -> Image:
```

Comment:
In my head that only strengthens the argument for `Image.create_dummy(idx: int = 0) -> Image`, as it would allow drop-in replacement with `Image.from_array(array: numpy.Array) -> Image` or `Image.from_beam(beam: SomeBeamDescription) -> Image`

Reply:
Makes sense, agreed

Reply:
When i try and make it a classmethod, what do I put as the return type? Putting Image says it isn't defined...

Reply:
Try "Image"